apps/web/src/components/Composer/Actions/Attachment.tsx

```diff
@@ -64,23 +64,44 @@ const Attachment: FC<Props> = ({ attachments, setAttachments }) => {
     return false;
   };
 
+  const isImageType = (files: any) => {
+    for (const file of files) {
+      if (!ALLOWED_IMAGE_TYPES.includes(file.type)) {
+        return false;
+      }
+    }
+
+    return true;
+  };
+
   const handleAttachment = async (evt: ChangeEvent<HTMLInputElement>) => {
     evt.preventDefault();
     setShowMenu(false);
     setLoading(true);
 
     try {
       // Count check
-      if (evt.target.files && (hasVideos(evt.target.files) || evt.target.files.length > 4)) {
-        return toast.error('Please choose either 1 video or up to 4 photos.');
+      if (
+        evt.target.files &&
+        (hasVideos(evt.target.files) ||
+          (isImageType(evt.target.files) && evt.target.files.length + attachments.length > 4))
```

Comment:
nit: assign `evt.target.files` to a const and use that

Reply:
@ercwangwh can you do this 🙇🏼

Reply:
@bigint updated 🙏